### PR TITLE
[4.1] scheduler sorting

### PR DESCRIPTION
**Pull request by brianteeman**
pr for #36210

This PR does multiple things

1. Changed the order of sort in the dropdown to match the column order of the table
2. Capitalised Task Type
3. Added sortable column to display the task priority

image

### CODE REVIEW

File: administrator/components/com_scheduler/tmpl/tasks/default.php [hunk 1]
```diff
@@ -231,6 +236,17 @@ class="js-draggable" data-url="<?php echo $saveOrderingUrl; ?>" data-direction="
 							</button>
 						</td>
 
+						<!-- Priority -->
+						<td class="small d-none d-lg-table-cell">
+							<?php if ($item->priority == '-1') : ?>
```
Comment (Quy) [inline administrator/components/com_scheduler/tmpl/tasks/default.php]: `$item->priority` is an integer.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 1ab0ae6b96e40d056dcaaccd6126812cc194401c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37224">issues.joomla.org/tracker/joomla-cms/37224</a>.</sub>

**Comment by MUX-ON-WINDOWS** [comment]
I have tested this item :white_check_mark: successfully on 1ab0ae6b96e40d056dcaaccd6126812cc194401c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37224">issues.joomla.org/tracker/joomla-cms/37224</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37224">issues.joomla.org/tracker/joomla-cms/37224</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
thanks

**Pull request merged**